Azure/sonic-buildimage
# [Mellanox] Update SAI to version 1.14.3

* [SAI] Add PFC pause duration counters in microseconds

Signed-off-by: Volodymyr Samotiy <volodymyrs@nvidia.com>



**- Why I did it**
To add PFC pause duration counters in microseconds
**- How I did it**
Updated SAI to version 1.14.3
**- How to verify it**

**- Which release branch to backport (provide reason below if selected)**
Build an image and check that PFC duration counters in microseconds are available in counters DB.



**- Description for the changelog**

[Mellanox] Update SAI to version 1.14.3
